### PR TITLE
yabai: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/osx/yabai.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# yabai
+
+> Un administrador de ventanas en mosaico para macOS basado en la partición de espacio binario.
+> Más información: <https://github.com/koekeishiya/yabai>.
+
+- Establecer el diseño a bsp:
```
Comment (navarroaxel) [inline pages.es/osx/yabai.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Establece la diposición a bsp:
```
Not sure what this command does, but for layour sometimes `disposición` (like "la disposición de los objetos en la sala") fits better

File: pages.es/osx/yabai.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`yabai -m config layout {{bsp}}`
+
+- Establecer el espacio de la ventana en 10pt:
```
Comment (navarroaxel) [inline pages.es/osx/yabai.md:10]:
```suggestion
- Establece el espacio de la ventana en 10pt:
```

File: pages.es/osx/yabai.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`yabai -m config window_gap {{10}}`
+
+- Habilitar opacidad:
```
Comment (navarroaxel) [inline pages.es/osx/yabai.md:14]:
```suggestion
- Habilita opacidad:
```

File: pages.es/osx/yabai.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`yabai -m config window_opacity on`
+
+- Deshabilitar sombra de la ventana:
```
Comment (navarroaxel) [inline pages.es/osx/yabai.md:18]:
```suggestion
- Deshabilita la sombra de la ventana:
```

File: pages.es/osx/yabai.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`yabai -m config window_shadow off`
+
+- Habilitar barra de estado:
```
Comment (navarroaxel) [inline pages.es/osx/yabai.md:22]:
```suggestion
- Habilita la barra de estado:
```

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**